### DISCUSSION TITLE
Add "Fork" information and filter

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#19] [New Feature Requests]
As an auditor, I would like to review findings from forks of other major protocols, so that it helps me understand what are the most common mistakes developer make when creating a fork. 

For example, I would like to filter by forks of Uniswap v2 and do that analysis.

**Comment by nisenkod** [comment]
good idea

**Reply by marr1inaa** [reply]
Very good

**Comment by huesos3214321** [comment]
big brain move i think

**Comment by peterpeterson5533** [comment]
Nice idea ))

**Reply by lobangala** [reply]
yes

**Comment by fedor4433fg** [comment]
I support this idea

**Comment by ingahola** [comment]
Idea is top

**Comment by aviggiano** [comment]
I'm amazed that this suggestion got so much attention!
I'm personally much more interested in https://github.com/orgs/solodit/discussions/9, but I guess this is more useful to other people 😅

**Comment by sadlymad** [comment]
great idea bro! agree

**Comment by MargoMat** [comment]
lets go

**Comment by Danielopa** [comment]
good

**Comment by Viollasma** [comment]
Hrllo friends, good info

**Comment by Breadmaker1** [comment]
Awesome 👏

**Comment by OneMore0** [comment]
We will succeed!!!

**Comment by VitalikYa** [comment]
Nice idea!

**Comment by 2Crystal** [comment]
How often do developers make mistakes when (forking)

**Comment by Stillerben** [comment]
Im voting for this issue

**Comment by hans-solodit** [comment]
We appreciate all the votes received. We have introduced a new filtering option called "Forked From." However, we currently lack the necessary mapping for this feature. To address this, we have decided to make it open source, allowing the community to contribute.

In our open [content repository](https://github.com/solodit/solodit_content), you will find two Markdown files:

- [legacy_protocols.md](https://github.com/solodit/solodit_content/blob/main/legacy_protocols.md) contains a list of the main protocols that are frequently forked.
- [forked_protocols.md](https://github.com/solodit/solodit_content/blob/main/forked_protocols.md) includes the protocols currently available on Solodit.

We encourage you to fork these files and open a pull request (PR) if you would like to contribute. 
Feel free to participate and make the necessary additions or modifications.

**Comment by IAm0xAxe** [comment]
Cool Idea!

**Comment by hans-cyfrin** [comment]
Once again, my appreciation goes out to @aviggiano for the invaluable suggestion, and to all who voted. 
I'm marking this feature request as resolved.

**Comment by Little0011Dragon** [comment]
Let's check the links.

**Comment by LunaLi777** [comment]
Nice idea!

**Comment by Breadmaker1** [comment]
Very nice

**Reply by stepangromak1** [reply]
Good idea but it's a difficult task.

**Comment by Achyt04** [comment]
very good idea!

**Comment by Ellu39940** [comment]
Good idea!

**Comment by ushakovvanna** [comment]
it will be useful to exclude errors

**Comment by Csgett** [comment]
Very nice

**Comment by kalinkavita** [comment]
Yes, sure

**Comment by vikserov14** [comment]
Yeah not bad
